### PR TITLE
include cookiebanner

### PR DESCRIPTION
**Pull request by sandikbarr**
some relevant discussion on the [exchange PR on cookie banners](https://github.com/PRX/exchange.prx.org/pull/404)

### CODE REVIEW

File: src/index.pug [hunk 1]
```diff
@@ -11,6 +11,12 @@ html(lang='en')
       link(rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro")
     for script in scripts
       link(rel="preload", href=script, as="script")
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/cookie-banner/1.2.2/cookiebanner.min.js",
```
Comment (sandikbarr) [inline src/index.pug]: tried to install this in node_modules and include in CLI scripts[] but didnt take

**Comment by kookster** [comment]
@sandikbarr were you making more changes, or should we merge?

**Comment by sandikbarr** [comment]
@kookster sounds like @pkarman has an open question about the top positioning on exchange at least, not sure that applies here, otherwise done here

**Comment by kookster** [comment]
I'm pretty ok with having them on top, I feel like they blending into the background, and were too easily ignored at bottom (at least on exchange), so I think I am good to go with this.

**Pull request merged**